t/live-https.t

```diff
@@ -51,12 +51,15 @@ SKIP: for my $pass ( 1 .. 2 ) {
         Accept            => '*/*'
     );
 
-    my ( $code, $mess, %h ) = $s->read_response_headers;
+    my ( $code, $mess, %headers ) = $s->read_response_headers;
+
+    my %h = map { lc($_) => $headers{$_} } keys %headers;
+
     print "# ----------------------------\n";
     print "# $code $mess\n";
     for ( sort keys %h ) {
         print "# $_: $h{$_}\n";
-        if (/^Connection$/i && $h{$_} =~ /^keep-alive$/) {
+        if (/^connection$/i && $h{$_} =~ /^keep-alive$/) {
```

Comment:
I think you can now skip the regular expression here withe the `/i` switch, and actually use a `eq`, but `/.../` works nice on `$_`

Reply:
Thanks @vanHoesel! I've removed the `i` in a subsequent commit.